deeds-dapp-webapp/src/main/webapp/static/css/deeds.css

```diff
@@ -383,3 +382,9 @@ h4 {
 .v-application.theme--dark .link--color {
   color: #4e9fff !important;
 }
+
+@media screen and (min-width: 1280px) {
```

Comment:
Use breakpoints from `platform-ui` (as done in other addons, import the `variables.less`)

Reply:
ok, no need for now to change it since it requires an advanced maven configuration to reuse `platform-ui`. you can keep it as it is until we make the integration in a proper task or when another item requires variables from `platform-ui` again.